VeterinaryValeriya/ClinicRegistry.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace VeterinaryValeriya
+{
+    class ClinicRegistry
+    {
+        private static Dictionary<int, DomesticAnimal> registry;
+
+        private static int countOfAnimals = 0;
```

Comment:
it should be better called something like nextUniqueSequentialNumber because used only for this purpose